pkg/annotation/k8s.go

```diff
@@ -105,6 +105,10 @@ const (
 	SharedService      = ServicePrefix + "/shared"
 	SharedServiceAlias = Prefix + "/shared-service"
 
+	// MCSAPIServiceSpec is an internal annotation to share MCS-API spec to
+	// facilitate the synchronization of MCS ServiceImport across clusters.
+	MCSAPIServiceSpec = ServicePrefix + "/mcs-api-internal-svc-spec"
```

Comment:
The suggestion sounds reasonable to me. That said, as you also mentioned below, it is not clear to me at the moment why we need this annotation, rather than having the appropriate controller looking up the corresponding `ServiceImport` object. Storing JSON content in an annotation feels pretty fragile to me.